hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -863,16 +883,17 @@ public FileStatus getFileStatus(final Path path) throws IOException {
 
     // generate continuation token if a valid startFrom is provided.
     if (startFrom != null && !startFrom.isEmpty()) {
-      continuation = getIsNamespaceEnabled()
+      continuation = getIsNamespaceEnabled(originalTracingContext)
               ? generateContinuationTokenForXns(startFrom)
               : generateContinuationTokenForNonXns(relativePath, startFrom);
     }
+    TracingContext tracingContext = new TracingContext(originalTracingContext);
```

Comment:
Why this ? Same tracingContext should be used as the calls are sequential.

Reply:
Removed